incubator/catalog-schema-registry/src/main/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/SchemaRegistryCatalogHandler.java

```diff
@@ -68,25 +84,27 @@ public String resolve(
     private String sendHttpRequest(
         String path)
     {
-        try
-        {
-            HttpRequest httpRequest = HttpRequest
+        HttpRequest httpRequest = HttpRequest
                 .newBuilder(toURI(baseUrl, path))
                 .GET()
                 .build();
-            // TODO: introduce interrupt/timeout for request to schema registry
+        // TODO: introduce interrupt/timeout for request to schema registry
+
+        try
+        {
             HttpResponse<String> response = client.send(httpRequest, HttpResponse.BodyHandlers.ofString());
             return response.statusCode() == 200 ? response.body() : null;
         }
         catch (Exception e)
         {
-            System.out.println("Error fetching schema");
+            System.out.println("Error fetching schema" + e);
         }
         return null;
     }
 
     private URI toURI(
-        String baseUrl, String path)
+        String baseUrl,
+        String path)
```

Comment:
Do we need to use `URI.create(baseUrl).resolve(path)` to address potential issue of missing `/` between baseURL and path during string concatenation?